src/libsystemd-network/icmp6-util.c

```diff
@@ -186,7 +186,6 @@ int icmp6_receive(int fd, void *buffer, size_t size, struct in6_addr *ret_dst,
 
         /* namelen == 0 only happens when running the test-suite over a socketpair */
 
-        assert(!(msg.msg_flags & MSG_CTRUNC));
```

Comment:
why drop this?

Reply:
`recvmsg_safe()` already handled the case.
You dropped the same check in resolved by 3691bcf3c5eebdcca5b4f1c51c745441c57a6cd1.

Reply:
indeed!